.isort.cfg

```diff
@@ -2,12 +2,13 @@
 ; see https://github.com/psf/black
 multi_line_output=3
 include_trailing_comma=True
-force_grid_wrap=0
```

Comment:
Is it a change we really need, if so, we need to update all the repos with it.

Reply:
Yeah I'll revert this, seems that this does not help to fix the problem with black and isort. That's why the fix I put was to put the skip on the import.

Reply:
ok

Reply:
So revert also all the changes in the other files that were not needed.

Reply:
Yeah I fixed that.